scss/components/_tabs.scss

```diff
@@ -138,7 +138,7 @@ $tab-content-padding: 1rem !default;
   display: none;
   padding: $padding;
 
-  &[aria-hidden="false"] {
+  &[aria-hidden=""] {
```

Comment:
wouldn't `&[aria-hidden]` be sufficient? It would just query for existence of the attribute